src/pages/categories/Categories.jsx

```diff
@@ -1,11 +1,19 @@
 import NoResultsFound from '~/components/no-results-found/NoResultsFound'
 import PageWrapper from '~/components/page-wrapper/PageWrapper'
 import CategoriesList from '~/components/categories-list/CategoriesList'
+import { useState, useEffect } from 'react'
+import useUrlSearchParams from '~/hooks/use-url-search-params'
 
 const Categories = () => {
+  const { searchParams, setUrlSearchParams } = useUrlSearchParams()
```

Comment:
move it in `CategoriesList` component.

Reply:
Thank you for your comment, I transferred the logic of the useUrlSearchParams() hook usage into CategoriesList